java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_1/Musician.java

```diff
@@ -0,0 +1,40 @@
+package com.zoolatech.lecture3.tasks._1;
+
+import java.util.Objects;
+
+public class Musician {
+    private String name;
+    MusicalInstrument instrument;
+
+    public Musician(String name, MusicalInstrument instrument) {
+        this.name = name;
+        this.instrument = instrument;
+    }
+
+    public void getNameAndInstrument(Musician musician) {
```

Comment:
```
A musician should also have a method which should print his/her name and the name of an instrument.
```

The idea was to print a current musician. In other words, it was enough to provide a method `toString`